script/__snapshots__/migrate-test-e2e.test.js.snap

```diff
@@ -0,0 +1,876 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`verify .all-contributorsrc 1`] = `
+"diff --git a/.all-contributorsrc b/.all-contributorsrc
+index 084b7ee..0ed629e 100644
```

Comment:
> `084b7ee..0ed629e`

Having the git commit hash contained in these snapshots means they'll be out of date whenever there's a new commit. I think they'll need to be stripped out.

Reply:
bump @Aslemammad, this is still a thing I think?

Reply:
oh sorry, i haven't seen this, let me work on it just right now.